app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<%=render "shared/navbar", right_el: "menu", left_el: "none", title: "TRANSACTIONS"%>
+<div class="bg-secondary text-light"><span>Total Amount: $100, Total Items: 9</span></div>
+    <ul class="category-list d-flex flex-column align-items-center m-auto  p-3">
+        <%for a in 0..30 do%>
+            <%= render "transactions/shared/transaction"%>
+        <%end%>
+    </ul>
```

Comment:
- [ ] [Optional] This change was missed by the previous reviewer so I'm marking it as optional but I would highly recommend to implementing it for a better user experience. 
I would suggest in the transaction page to show the total amount at the top of the page as requested in the project requirements, Currently it's showing at the bottom of the page.

![image](https://github.com/nilab-khwaja/budget-app/assets/83803414/e273ac5d-163c-4627-95ec-3fb5311bcb76)